### PR TITLE
[12.0][ADD] iot_input

### PR DESCRIPTION
**Pull request by etobella**
This addon allows to use a device in order to input data to odoo automatically.

It opens a URL that a device can use to connect (with a password) that can only
execute an specific action.

Inputs are useful when a device wants to communicate to odoo for a single
and simple action.
This way, the device does not need to be configured with a odoo user and
password, it is handled by odoo devices.

Examples:

* Sending the temperature every three minutes.
* Sending the RFID that the device has received in order to perform some action

@jarroyomorales @HviorForgeFlow @LoisRForgeFlow

### CODE REVIEW

**Comment by flotho** [comment]
Hi,

This PR looks very interesting. 
I read carefully the code and first I wasn't  able to identify if this module is supposed to be autonomous or if it's a base for other modules. I think that you should add more info in read me to explain that :
* Out of the box this module helps to execute all the method already exsiting in Odoo. This can be found there https://github.com/OCA/iot/pull/12/files#diff-01d1ac3fe4df459e22982bf5c0f268f9R12
* Any other developpement don't need to reference this module to be used for IoT Purpose

Moreover, from my opinion, this also could be usefull for other usages that are not coming from IoT devices but by other applications.
Maybe if we consider this other usage, this moule could be proposed in server tools as a base for multiple usages?

Anyway, really good job there,

Regards

File: iot_input/README.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+This addon allows to use a device in order to input data to odoo automatically.
+
+It opens a URL that a device can use to connect (with a password) that can only
+execute an specific action.
+
+Inputs are useful when a device wants to communicate to odoo for a single
+and simple action.
+This way, the device does not need to be configured with a odoo user and
+password, it is handled by odoo devices.
+
```
Comment (flotho) [inline iot_input/README.rst:37]: Hi,
more comments could be usefull.

**Review by flotho — APPROVED** [review]

**Comment by etobella** [comment]
Thanks @flotho I will add more comments to the module description.
We have been working with this module for half a year as an Attendance System and Opening door system using RFID as identifiers. It can be used in other cases outside IoT, but the main point of this module is to create a single entrance for an external tool that can only do a single action. For this reason we choose IoT. The main intent is for simple instructions, so IoT was the main choice.

**Review by jarroyomorales — APPROVED** [review]
Tested and works as expected! :+1:

**Pull request merged**